index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <header class="header">
+      <a class="mylogo">My Logo</a>
+      <nav class='nav'>
+        <img src='images/hamburg.svg' alt='' title=''>
+      </nav>
+    </header>
+    <section>
+     	<h1 class="myheader">I’m PRATAP Glad to see you!</h1>
+    	<h4 class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+    	<a class="connect">LET’S CONNECT</a>
```

Comment:
- [optional] kindly use a `hx` here as this is a small heading to the social icons